dev-python/PyQtWebEngine/PyQtWebEngine-5.12.1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+inherit python-r1 qmake-utils
+
+DESCRIPTION="Python bindings for Qt WebEngine framework"
+HOMEPAGE="https://www.riverbankcomputing.com/software/pyqtwebengine/intro"
+
+MY_P=${PN}_gpl-${PV/_pre/.dev}
+SRC_URI="https://www.riverbankcomputing.com/static/Downloads/${PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-python/PyQt5-5.12.1[${PYTHON_USEDEP},widgets]
+	>=dev-python/PyQt5-sip-4.19.14:=[${PYTHON_USEDEP}]
+	>=dev-qt/qtwebengine-5.12[widgets]"
+DEPEND="${RDEPEND}
+	>=dev-python/sip-4.19.14[${PYTHON_USEDEP}]"
+BDEPEND=""
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	default
+	python_copy_sources
+}
+
+src_configure() {
+	configuration() {
+		local myconf=(
+			"${PYTHON}"
+			"${S}"/configure.py
+			$(usex debug '--debug --trace' '')
+			--destdir="$(python_get_sitedir)"/PyQt5
+			--pyqt-sipdir="$(python_get_scriptdir)"/../../../share/sip/PyQt5
+			--qmake="$(qt5_get_bindir)"/qmake
+			--sipdir="$(python_get_scriptdir)"/../../../share/sip/PyQt5
+			--sip-incdir="$(python_get_includedir)"
+			--sysroot="${ESYSROOT}/usr"
```

Comment:
have you tested this? I doubt all dependencies are ready, so I'd say don't bother passing `--sysroot=`

Reply:
No, I did not test installing PyQtWebEngine on a prefixed system, there are just too many (and some circular) dependencies, indeed. But if all of the dependencies support prefix (they should do it sooner or later, right?), then why should not we support prefix in `dev-python/PyQtWebEngine`? Even if some of the dependencies do not yet work with prefixes, are there any drawbacks in setting `--sysroot` here?

Reply:
I shall leave it without setting before I or someone else will test it the package a prefix. Also now I do not recall why did I write "/usr" here... must be a mistake.